app/src/main/java/org/simple/clinic/summary/teleconsultation/sync/TeleconsultFacilityInfoApi.kt

```diff
@@ -6,6 +6,6 @@ import retrofit2.http.Headers
 
 interface TeleconsultFacilityInfoApi {
   @Headers(value = ["X-RESYNC-TOKEN: 1"])
-  @GET("v4/teleconsultation_medical_officers/sync")
+  @GET("v4/facility_medical_officers/sync")
```

Comment:
Is this available in QA? If yes, we probably should also enable the sync for it since it's currently disabled.

Reply:
Not sure, let me check with server devs and enable it if it's available.

Reply:
When you enable it, there's also a sync integration test that we will need to enable again.